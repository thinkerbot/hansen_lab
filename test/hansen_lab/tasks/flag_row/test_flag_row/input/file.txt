a	true
b	false
c	true