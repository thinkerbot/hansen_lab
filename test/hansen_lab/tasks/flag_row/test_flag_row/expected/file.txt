a	true	1
b	false	0
c	true	1